.github/workflows/tests.yml

```diff
@@ -48,10 +48,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python: 2.7
+          - python: 3.6
           - python: 3.7
           - python: 3.8
-          - python: 3.9.1
+          - python: 3.9
+          - python: '3.10.0-alpha - 3.10' # Latest Python version
```

Comment:
see my proposed changes in #470
I'm not sure we have to always test with the newest Python releases, especially if they're alpha, beta, rc., etc.

Reply:
I think you are right. Deleting this.